sisl/io/siesta/ani.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+Sile object for reading ANI files
+"""
+import numpy as np
+
+from .sile import SileSiesta
+from ..sile import add_sile, sile_fh_open
+from sisl._internal import set_module
+from sisl import Geometry, SuperCell
+
+__all__ = ["aniSileSiesta"]
+
+@set_module("sisl.io.siesta")
+class aniSileSiesta(SileSiesta):
+    """ ANI file object """
+
+    @sile_fh_open(True)
+    def read_geometry(self, step=1):
```

Comment:
Could we perhaps have a `start=0, step=1` argument to allow one to start arbitrarily?
Could some value (`none` or negative) be specified to only retrieve 1 of the geometries?

`read_geometry(start=200, step=None) == Geometry` (not a list)

Reply:
I think this is now enabled.